### PR TITLE
Added whiteout for pause feature

### PR DESCRIPTION
**Pull request by Brent1798**

### CODE REVIEW

**Comment by Brent1798** [comment]
Things wrong:
Doesn't re-size with page.
Doesn't close with resume of game.

File: index.html [hunk 1]
```diff
@@ -34,7 +38,7 @@
       
       // initiate game at Boot state
       game.state.start('Boot');
-    }
+    };
   </script>
 </body>
-</html>
+</html>
```
Comment (jcandan) [inline index.html]: it's best practice to leave a return at the bottom of all documents.

File: js/Game.js [hunk 2]
```diff
@@ -65,6 +65,8 @@ Critterer.Game.prototype = {
     },
     
     create: function () {
+        // Set up!
```
Comment (jcandan) [inline js/Game.js]: erroneous comment should be removed.

File: js/Game.js [hunk 3]
```diff
@@ -104,6 +106,11 @@ Critterer.Game.prototype = {
        
     },
     
+    //white out page
+    whiteout: function(){
```
Comment (jcandan) [inline js/Game.js]: This is a blank function, is it necessary?

File: js/MainMenu.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ Critterer.MainMenu.prototype = {
          * Implements create()
          */
         create: function() {
-
+            this.fgColor= "red";
```
Comment (jcandan) [inline js/MainMenu.js]: This and the test.html document should be removed.

**Comment by jcandan** [comment]
Please pull the latest from `upstream` and push this merge up to your repo.

File: index.html [hunk 5]
```diff
@@ -18,6 +18,10 @@
     
 </head>
 <body>
+
```
Comment (jcandan) [inline index.html]: this is not necessary. you should be able to draw a rectangle object within Phaser and assign a width and height equal to the browser (e.g. `this.game.width`).

check out the following for help:
```
var PausePanel = function(game, parent){
...
    var backgroundColor = "0xffffff";
    var backgroundOpacity = 0.7;

    var pause_bg = game.add.graphics(game.width, game.height);
    pause_bg.beginFill(backgroundColor, backgroundOpacity);
    pause_bg.x = 0;
    pause_bg.y = 0;

    pause_bg.drawRect(0, 0, game.width, game.height);

    this.add(pause_bg);
```

**Comment by jcandan** [comment]
I know this is a lot of comments. try to commit more often. each of these changes should probably have been a commit.

**Comment by jcandan** [comment]
no changes for a while. closing as outdated.

**Pull request closed without merging**